ngraph/test/onnx/onnx_import.in.cpp

```diff
@@ -2309,107 +2313,107 @@ NGRAPH_TEST(${BACKEND_NAME}, onnx_model_scatter_elements_import_only)
-    test_case.run();
-}
-
-NGRAPH_TEST(${BACKEND_NAME}, onnx_upsample8_linear_infer)
-{
-    const auto function = onnx_import::import_onnx_model(
-        file_util::path_join(SERIALIZED_ZOO, "onnx/upsample8_linear.prototxt"));
-
-    // Input data shape (1, 1, 2, 2)
-    // Scales attribute values {1.0, 1.0, 2.0, 2.0}
-    // mode: linear
-
-    const Shape expected_output_shape{1, 1, 4, 4};
-    auto test_case = test::TestCase<TestEngine>(function);
-    test_case.add_input<float>({1.0, 2.0, 3.0, 4.0});
-    test_case.add_expected_output<float>(
-        expected_output_shape,
-        {1.0, 1.5, 2.0, 2.0, 2.0, 2.5, 3.0, 3.0, 3.0, 3.5, 4.0, 4.0, 3.0, 3.5, 4.0, 4.0});
-    test_case.run();
-}
-
-NGRAPH_TEST(${BACKEND_NAME}, onnx_upsample9_scales_const_import_only)
-{
-    const auto function = onnx_import::import_onnx_model(
-        file_util::path_join(SERIALIZED_ZOO, "onnx/upsample9_scales_const_nearest.prototxt"));
-
-    // Input data shape (1, 1, 2, 2)
-    // Input const scales values {1.0, 1.0, 2.0, 3.0}
-
-    const Shape expected_output_shape{1, 1, 4, 6};
-    EXPECT_EQ(function->get_output_size(), 1);
-    EXPECT_EQ(function->get_output_shape(0), expected_output_shape);
-    EXPECT_EQ(count_ops_of_type<onnx_import::default_opset::Interpolate>(function), 1);
-    EXPECT_EQ(count_ops_of_type<onnx_import::default_opset::Constant>(function), 1);
-}
-
-NGRAPH_TEST(${BACKEND_NAME}, onnx_upsample9_scales_const_nearest_infer)
-{
-    const auto function = onnx_import::import_onnx_model(
-        file_util::path_join(SERIALIZED_ZOO, "onnx/upsample9_scales_const_nearest.prototxt"));
-
-    // Input data shape (1, 1, 2, 2)
-    // Input const scales values {1.0, 1.0, 2.0, 3.0}
-    // mode: nearest
-
-    const Shape expected_output_shape{1, 1, 4, 6};
-    auto test_case = test::TestCase<TestEngine>(function);
-    test_case.add_input<float>({1.0, 2.0, 3.0, 4.0});
-    test_case.add_expected_output<float>(
-        expected_output_shape, {1.0, 1.0, 1.0, 2.0, 2.0, 2.0, 1.0, 1.0, 1.0, 2.0, 2.0, 2.0,
-                                3.0, 3.0, 3.0, 4.0, 4.0, 4.0, 3.0, 3.0, 3.0, 4.0, 4.0, 4.0});
-    test_case.run();
-}
-
-NGRAPH_TEST(${BACKEND_NAME}, onnx_upsample9_scales_const_linear_infer)
-{
-    const auto function = onnx_import::import_onnx_model(
-        file_util::path_join(SERIALIZED_ZOO, "onnx/upsample9_scales_const_linear.prototxt"));
-
-    // Input data shape (1, 1, 2, 2)
-    // Input const scales values {1.0, 1.0, 2.0, 2.0}
-    // mode: linear
-
-    const Shape expected_output_shape{1, 1, 4, 4};
-    auto test_case = test::TestCase<TestEngine>(function);
-    test_case.add_input<float>({1.0, 2.0, 3.0, 4.0});
-    test_case.add_expected_output<float>(
-        expected_output_shape,
-        {1.0, 1.5, 2.0, 2.0, 2.0, 2.5, 3.0, 3.0, 3.0, 3.5, 4.0, 4.0, 3.0, 3.5, 4.0, 4.0});
-    test_case.run();
-}
+// NGRAPH_TEST(${BACKEND_NAME}, onnx_upsample8_import_only)
```

Comment:
What about these tests? Why did you comment it? Is it regression?

Reply:
Now I debug them.

Reply:
Please resolve this issue before the merge.

Reply:
Why we introduced a regression but don't want to fix it in this PR?
Looks like we want to merge this PR without documentation and with a regression in tests...

In any way we shouldn't comment these tests. If it is not work on CPU, we can use manifest file in order to disabled them.

Reply:
There are several other PRs depending on this one, so we want to merge this ASAP.

Reply:
In this case, we need create a tasks to fix all questions for this PR, like documentation, tests and etc.

Reply:
Tests are enabled.